src/form.js

```diff
@@ -844,7 +864,7 @@ define([
         },
         _updateMugPath: function (mug, oldHashtag, newHashtag) {
             var map = this.mugMap, newPath;
-            delete map[this.normalizeHashtag(oldHashtag)];
+            delete map[this.normalizeEscapedHashtag(oldHashtag)];
```

Comment:
Why do the keys to `mugMap` need to be passed through `normalizedEscapedHashtag`? It seems like those keys are never invalid xpath, nor do they need to be converted to bubbles, which seem like the only reasons you'd want to use an escaped hashtag.